src/lib/components/ColorPicker.svelte

```diff
@@ -0,0 +1,87 @@
+<script lang="ts">
+	import { user, onlineUsers } from '$lib/stores/stateStore';
+	import { colors } from '../color';
+	import type { User, Color } from '$lib/stores/stateStore';
+
+	/* Checks if a color is chosen and returns true or false so the color-button gets disabled */
+	function colorChosen(color: Color, user: User, onlineUsers: User[]) {
+		if (user && user.color && user.color.name === color.name) {
```

Comment:
Hvad betyder det at "user.color" kan evaluere til en boolsk værdi?

Reply:
For den sags skyld, hvad det betyde det at en `user` evalueres til en boolsk værdi